EGO Forum Enhancement.user.js

```diff
@@ -296,11 +392,21 @@ function handleLeadership() {
     generateRedText("80%");
 }
 
+/**
+ * Changes the target of the application links to open in a new tab
+ * @returns void
+ */
+function handleApplicationPage() {
+    document.querySelector('.dataList-row > :nth-child(2)').childNodes[1].setAttribute('target', '_blank');
+    document.querySelector('.dataList-row > :nth-child(2)').childNodes[3].setAttribute('target', '_blank');
```

Comment:
Is it possible to adjust these to use a more lengthy querySelector as opposed to childNodes? The application buttons may be modified by another script, or even updated, which can cause childNodes to point to the wrong element. Same goes for `:nth-child` unless there's no other choice.